java/src/jmri/jmrit/throttle/ThrottlesTableModel.java

```diff
@@ -46,9 +47,6 @@ public void addThrottleFrame(ThrottleFrame tf) {
 
     public void removeThrottleFrame(ThrottleFrame tf, DccLocoAddress la) {
         throttleFrames.remove(tf);
-        if (la != null) {
-            jmri.InstanceManager.throttleManagerInstance().removeListener(la, this);
-        }
```

Comment:
An explanation to this change:
`ThrottlesTableModel` never registers a listener to the throttle manager. It registers a property change listener to the throttle, but the throttle might be used for several throttle frames so it doesn't work to unregister the listener to the throttle. But I don't think that matters.